controllers/url.js

```diff
@@ -4,6 +4,7 @@ const config = require('config');
 const redis = require('redis');
 
 const { generateHash } = require('../utils');
+const { urlencoded } = require('body-parser');
```

Comment:
Why using it here. we are already using it as a middleware in the index.js file

Reply:
Yeh Made it by-mistake..

Reply:
no need here